### DISCUSSION TITLE
Persona with custom presence icons?

### DISCUSSION THREAD

**Discussion by GordonSmith** [microsoft/fluentui#17214] [Q&A] [answered]
Are there any examples of using Persona with custom Presence Icons?

**Comment by TheColdPot** [comment, marked as answer]
Persona has a `presence` prop. It's type is enum `PersonaPresence`.
If you want to add an icon with a presence,
```tsx
<Persona
    {...OtherPersonaProps}
    size={PersonaSize.size8}
    presence={PersonaPresence.online}
 />
```

`PersonaPresence` has following values:
```
away     = 3
blocked  = 5
busy     = 6
dnd      = 4
none     = 0
offline  = 1
online   = 2
```
Fluent UI cannot having custom image for presence yet. You can submit an issue or raise an idea.
